File: src/rules.py
from __future__ import annotations

from dataclasses import dataclass


@dataclass(frozen=True)
class MissionResult:
    correct: bool
    feedback: str


def answer_municipal_health_mission(answer: str) -> MissionResult:
    if answer == "A":
        return MissionResult(
            True,
            "Correto. O prefeito administra servicos municipais; a Camara cria leis locais e fiscaliza.",
        )
    return MissionResult(
        False,
        "Ainda nao. Pense no nivel de governo responsavel por servicos da cidade. Tente novamente.",
    )

File: web/src/game/rules.ts
// Portado de src/rules.py

export interface MissionResult {
  correct: boolean;
  feedback: string;
}

export function answerMunicipalHealthMission(answer: string): MissionResult {
  if (answer === "A") {
    return {
      correct: true,
      feedback:
        "Correto. O prefeito administra servicos municipais; a Camara cria leis locais e fiscaliza.",
    };
  }
  return {
    correct: false,
    feedback:
      "Ainda nao. Pense no nivel de governo responsavel por servicos da cidade. Tente novamente.",
  };
}
